src/util.h

```diff
@@ -44,6 +44,9 @@ class CTranslationInterface
     boost::signals2::signal<std::string (const char* psz)> Translate;
 };
 
+extern const int HELP_MESSAGE_FILTER_NONE;
```

Comment:
Why not define the flags in the header? That's the usual approach used for constants throughout the code, and has the advantage of making the values available to the compiler so the linker doesn't have to reserve memory for them.